### DISCUSSION TITLE
Reduce padding in Paste Tables

### DISCUSSION THREAD

**Discussion by ldvlgr** [twilio-labs/paste#3495] [General] [answered]
Is there a way to reduce the padding in Paste tables?  It appears that it always has `padding 12px 16px` on each table cell.
I would like to use Paste tables in Flex that are more condensed (less white space) to be able to display more data.
No luck with: `<Th style={{ padding: '4px' }}>`
Is there are better way to override the padding?

**Comment by serifluous** [comment, marked as answer]
Hey @ldvlgr ! From your screenshot in Slack, it seems like you might be building a Flex plugin for customers. If that's the case, you can use the customization provider to change the padding: https://paste.twilio.design/customization

However if this is not a customer-specific implementation and it's a "native" Twilio experience instead, we don't allow components to be customized. But let us know if this is your scenario, though.

**Comment by ldvlgr** [comment]
Thanks @serifluous Yes, I found the way to customize the Td/Th using the [element](https://paste.twilio.design/customization/customizing-component-elements#understand-the-element-prop-and-data-attributes) attribute.
```
COMPACT_TABLE: {
    padding: 'space20',
  },
```
  and then
  `<Th element="COMPACT_TABLE">`
